nuxeo-java-client/src/main/java/org/nuxeo/client/objects/blob/Blob.java

```diff
@@ -33,6 +33,11 @@
 
     int getLength();
 
+    /**
+     * @since 3.1
+     */
+    long getLongLength();
```

Comment:
hmm, a bit messy to put the type in the method name.  I don't have a better solution, other than changing the return type of `getLength`!  Is `getLength` deprecated?

Reply:
How about `getContentLength`?

Reply:
I should deprecate `getLength` indeed, agreed for `getContentLength`